open_lm/main.py

```diff
@@ -399,7 +399,11 @@ def main(args):
     if args.hf_model is not None:
         model = create_wrapped_hf_model(args)
     else:
-        with torch.device("meta" if args.fsdp else args.device):
+        # with torch.device("meta" if args.fsdp else args.device):
+        #     model = create_model(args)
+        # if not args.fsdp:
+        #     model.reset_parameters()
+        with torch.device(args.device):
```

Comment:
Note for later: before merging this pr, let's merge https://github.com/mlfoundations/open_lm/pull/155 and require --disable-meta-device for MoE

Reply:
this is done!